server/src/test/java/org/candlepin/controller/PoolManagerFunctionalTest.java

```diff
@@ -207,7 +209,7 @@ public void testQuantityCheck() throws Exception {
 
         // The cert should specify 5 monitoring entitlements, taking a 6th should fail:
         assertEquals(null, poolManager.entitleByProducts(data));
-
+        monitoringPool = poolCurator.find(monitoringPool.getId());
```

Comment:
@nguyenfilip needed to add this when I was clearing the cache at some point of time. will be cleaned up in the merged PR.